MediaBrowser.Api/Sessions/SessionService.cs

```diff
@@ -254,26 +245,10 @@ public class GetPasswordResetProviders : IReturn<NameIdPair[]>
     {
     }
 
-    [Route("/Auth/Keys/{Key}", "DELETE")]
-    [Authenticated(Roles = "Admin")]
-    public class RevokeKey
-    {
-        [ApiMember(Name = "Key", Description = "Auth Key", IsRequired = true, DataType = "string", ParameterType = "path", Verb = "DELETE")]
-        public string Key { get; set; }
-    }
-
-    [Route("/Auth/Keys", "POST")]
-    [Authenticated(Roles = "Admin")]
-    public class CreateKey
-    {
-        [ApiMember(Name = "App", Description = "App", IsRequired = true, DataType = "string", ParameterType = "query", Verb = "POST")]
-        public string App { get; set; }
-    }
-
     /// <summary>
     /// Class SessionsService.
```

Comment:
Fix

Reply:
Those are littered throughout the project, and we definitely have more useless comments in other places. I'd be fine removing them all but I'm not sure if that will help or hinder documentation progress.

Reply:
Leave it it for now, otherwise it will just add to the pile of warnings